CHANGES.md

```diff
@@ -14,7 +14,26 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- Various changes to get the new production host up and running
+
+    **Non-breaking changes**
+    - Boostrap testsuite: only crawl the subset enough to pass canarie-api monitoring: faster when system under test has too much other stuff.
```

Comment:
`Boostrap` in here seems more to be a kind of music than a stack deployment concept ;)

Reply:
I meant "bootstrap" LOL, typo :D